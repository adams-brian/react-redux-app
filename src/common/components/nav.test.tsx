import { shallow } from 'enzyme';
import * as React from 'react';

import Nav from './nav';

describe("Nav", () => {

  it('renders as expected', () => {
    const component = shallow(<Nav />);
    expect(component).toMatchSnapshot();
  });
  
});
