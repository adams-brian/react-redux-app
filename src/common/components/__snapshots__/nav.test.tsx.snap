// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders as expected 1`] = `
<div>
  <nav
    className="navbar navbar-expand-lg navbar-dark bg-dark"
  >
    <button
      aria-controls="navbarNav"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarNav"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarNav"
    >
      <ul
        className="navbar-nav"
      >
        <li
          className="nav-item"
        >
          <Link
            className="nav-link"
            replace={false}
            to="/counters"
          >
            Counters
          </Link>
        </li>
        <li
          className="nav-item"
        >
          <Link
            className="nav-link"
            replace={false}
            to="/users"
          >
            Users
          </Link>
        </li>
        <li
          className="nav-item"
        >
          <Link
            className="nav-link"
            replace={false}
            to="/about"
          >
            About
          </Link>
        </li>
      </ul>
    </div>
  </nav>
</div>
`;
